workshops/util.py

```diff
@@ -476,113 +476,113 @@ def generate_url_to_event_index(website_url):
     raise WrongWorkshopURL()
 
 
-def find_tags_on_event_index(content):
-    """Given workshop's raw `index.html`, find and take YAML tags that
+def find_metadata_on_event_index(content):
```

Comment:
I would change this to `find_metadta_on_event_homepage`, since the word "index" can mean many things (e.g., a database index). Very small point...